sdk/keyvault/keyvault-admin/test/internal/userAgent.spec.ts

```diff
@@ -16,7 +17,32 @@ describe("Key Vault Admin's user agent (only in Node, because of fs)", function
   });
 
   it("SDK_VERSION and packageVersion should match", async function () {
-    assert.equal(SDK_VERSION, packageVersion);
+    let userAgent: string | undefined;
+    const client = new KeyVaultAccessControlClient(
+      "https://myvault.vault.azure.net",
+      {} as TokenCredential,
+      {
+        additionalPolicies: [
+          {
+            policy: {
+              name: "fetchUserAgent",
+              sendRequest: (request, next) => {
+                userAgent = request.headers.get("user-agent");
+                return next(request);
+              },
+            },
+            position: "perCall",
+          },
+        ],
```

Comment:
That works great! Thanks 😄 it also allows me to update all 4 packages today instead of relying on custom policies - double thanks! 

I'll update the 4 packages in this PR as well